content/geoip/docs/web-services/responses.md

```diff
@@ -0,0 +1,1231 @@
+---
+draft: false
+title: GeoIP and GeoLite API Responses
+---
+
+## Headers
+
+The `Content-Type` for a successful response varies based on the service as
+outlined below:
+
+{{< table >}}
```

Comment:
I noticed these wide tables cause the whole page to scroll rather than just the table itself. I think we'll want to preserve the current behavior where just the table is scrollable. What do you think?
<img width="491" alt="Screenshot 2024-07-19 at 8 53 21 AM" src="https://github.com/user-attachments/assets/60a765e4-7d0a-4ae1-84b9-83841d737cbe">

Reply:
Hmm... yeah, I thought the table scrolling behaviour carried over when the styles were migrated.  I'm also seeing different behaviour on my computer, where the whole viewport shrinks instead of page elements being responsive.

![image](https://github.com/user-attachments/assets/1d060ad9-6edd-4e4f-8fae-7715f7b54ad2)